webapp/src/apis/intersections/events-api.ts

```diff
@@ -60,9 +62,11 @@ class EventsApi {
       start_time_utc_millis: startTime.getTime().toString(),
       end_time_utc_millis: endTime.getTime().toString(),
     }
+    // if (roadRegulatorId) queryParams["road_regulator_id"] = roadRegulatorId;
 
     const events: MessageMonitor.Event[] = []
     for (const eventTypeObj of EVENT_TYPES) {
+      console.log(`Retrieving events of type ${eventTypeObj.value}`)
```

Comment:
Is this log worth leaving? I am not against leaving some console.log messages in.

Reply:
removed